doc/source/serve/advanced-guides/dev-workflow.md

```diff
@@ -79,6 +79,18 @@ After you're done testing, you can shut down Ray Serve by interrupting the `serv
 
 Note that rerunning `serve run` will redeploy all deployments. To prevent redeploying those deployments whose code hasn't changed, you can use `serve deploy`; see the [Production Guide](serve-in-production) for details.
 
+### Local Testing Mode
+
+Ray Serve now supports a local testing mode that allows you to run your deployments locally in a single process. This mode is useful for unit testing and debugging your application logic without the overhead of a full Ray cluster. To enable this mode, use the `_local_testing_mode` flag in the `serve.run` function:
+
+```python
+serve.run(app, _local_testing_mode=True)
```

Comment:
Kinda nit, but we should move this code into `local_dev.py` so it's tested correctly. You can follow this as an example to link the code back https://github.com/ray-project/ray/blob/17579cba9b793c2eccd2937d7a58e26ce22646db/doc/source/serve/advanced-guides/dev-workflow.md?plain=1#L31-L35